docs/modules/user-guide/partials/proc_adding-attributes-to-a-devfile.adoc

```diff
@@ -25,9 +25,9 @@ schemaVersion: 2.0.0
 metadata:
   name: petclinic-dev-environment
 components:
-  - name: myApp
+  - name: myapp
     kubernetes:
-      local: my-app.yaml
+      uri: my-app.yaml
 attributes:
```

Comment:
This doc seems to be adding the global attributes which is not present in the schema as of yet.

@elsony 

1. should this be updated to metadata.attributes?
2. the examples given here talk about use cases but they seem to suggest these are optional properties and not tool specific information, should they be updated that way?

Reply:
Looking at the specific documented attributes `editorFree`, it sounds like it is a well-known attribute with a pre-defined behaviour. However, that attribute is not part of the spec (I wonder where that info is from). +1 on we can't use that as an example yet given that we haven't introduced the global attribute yet. Even if we move the sample attribute under the `metadata` section, we should reword this example to make it clear that it is just defining a custom attribute.